src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/move_eltwise_up_data_movement.cpp

```diff
@@ -86,12 +90,12 @@ ov::intel_cpu::MoveEltwiseUpThroughDataMov::MoveEltwiseUpThroughDataMov() {
         }
 
         // eltwise constant shape should match new input shape
-        if (is_binary_op && current->get_output_partial_shape(0).rank().get_length() != eltwise->get_input_partial_shape(1).rank().get_length()) {
-            auto old_eltwise_const = std::dynamic_pointer_cast<ov::opset8::Constant>(eltwise->get_input_node_shared_ptr(1));
-            auto new_constant = std::make_shared<ov::opset8::Constant>(*old_eltwise_const.get(), ov::Shape{});
-
-            ov::copy_runtime_info(old_eltwise_const, new_constant);
-            ov::replace_node(old_eltwise_const, new_constant);
+        for (size_t i = 1; i < eltwise->get_input_size(); i++) {
+            if (current->get_output_partial_shape(0).size() != eltwise->get_input_partial_shape(i).size()) {
```

Comment:
Minor: actually we can skip constant replacement if the existing constant has empty shape already

Reply:
Add check for empty shape.